app/controllers/beta/project_boards_controller.rb

```diff
@@ -5,17 +5,20 @@ def show
     authorize current_user
 
     result = Beta::ProjectBoardOperations::Read.call(
-      params[:id],
       current_user,
-      current_flow: cookies[:current_flow],
-      projects_scope: policy_scope(Project)
+      project,
+      current_flow: cookies[:current_flow]
     )
 
     render json: result.data.as_json(root: false)
   end
 
   private
 
+  def project
+    @project = policy_scope(Project).friendly.find(params[:id])
```

Comment:
Should this be like this?

```suggestion
    @project ||= policy_scope(Project).friendly.find(params[:id])
```

Reply:
Hmm, I don't think so, because I have to update everytime, right?

Reply:
What do you mean? This will only be memoized for this instance, which is recreated for every request. Does it have to be updated inside the same instance?